src/locales/tr.json

```diff
@@ -231,6 +231,38 @@
     "aboutUsHeader": "Hakkımızda",
     "aboutUsText": "Teknolojiyi insanların yararına kullanmakta ortak paydasını bulan çok uluslu bir gençlik grubuyuz. Orta Doğu ve Kuzey Afrika'daki çocukları eğitici oyunlar aracılığıyla çevre sorunları ve iklim değişikliği konusunda eğitmek için birlikte çalışan 5 genç geliştirici ve 1 baş mühendisten oluşan bir grup. Hedef kitlemiz MENA'da yaşayan 8-13 yaş arası çocuklar; bu nedenle, web sitemizi çocukların ve öğretmenlerinin / ebeveynlerinin web sitesinde kolayca gezinmelerine yardımcı olmak için üç ana dilde sunuyoruz.",
     "whyWeDoItHeader": "Neden yapıyoruz",
-    "whyWeDoItText": "İnsanların doğayı ve özelliklerini korumak gibi ahlaki bir yükümlülüğü vardır, çünkü çevreyi korumak insanlığı toksik maddelerden ve gıda kalitemizi korur. MENA'daki yeni neslin, yerel ve küresel ölçekte insani ve sosyal konulara inanmak ve fayda sağlamak için hareket etmek için doğru araçlarla desteklenmesi gerektiğine inanıyoruz. Doğru araçlar verildiğinde, bir nesil aktivist yetiştirmek imkansız değildir."
+    "whyWeDoItText": "İnsanların doğayı ve özelliklerini korumak gibi ahlaki bir yükümlülüğü vardır, çünkü çevreyi korumak insanlığı toksik maddelerden ve gıda kalitemizi korur. MENA'daki yeni neslin, yerel ve küresel ölçekte insani ve sosyal konulara inanmak ve fayda sağlamak için hareket etmek için doğru araçlarla desteklenmesi gerektiğine inanıyoruz. Doğru araçlar verildiğinde, aktivist bir nesil yetiştirmek imkansız değildir.",
+    "teamInfo": {
+      "ammar": {
+        "name": "Ammar Sammour",
+        "bio": "",
+        "role": "Ekip Lideri"
+      },
+      "ibrahim": {
+        "name": "Ibrahim Aljarad",
+        "bio": "Selam, ben Suriye'den İbrahim, Flatiron kodlama okulunda kodlama okudum ve Alfurat Üniversitesi Ekonomi bölümünü tamamladım. Re:coded organizasyonunun Web Arayüz Geliştirme temel eğitim programına katıldım ve mezun oldum. Kodlama dışındaki diğer deneyimlerim: saha yapım ekiplerine teknik destek sağlayan uydu yapımcısı olarak çeşitli kanallar ve ajanslarla çalıştım.",
```

Comment:
i believe that arayüz i used only for "UI" and ön yüz for frontend, but don't trust me on this one :) a quick google search shows that ön yüz is usable to mean frontend though